__tests__/fixtures.ts

```diff
@@ -47,10 +47,7 @@ export const compiledC210 = readContractSierra('cairo/cairo210/cairo210.sierra')
 export const compiledC210Casm = readContractSierraCasm('cairo/cairo210/cairo210');
 
 export const getTestProvider = (): ProviderInterface => {
-  const provider = process.env.TEST_RPC_URL
-    ? new RpcProvider({ nodeUrl: process.env.TEST_RPC_URL })
-    : new SequencerProvider({ baseUrl: process.env.TEST_PROVIDER_BASE_URL || '' });
-
+  const provider = new RpcProvider({ nodeUrl: process.env.TEST_RPC_URL });
```

Comment:
Might be worth considering moving this change to a future PR so the Sequencer tests can pass (one last time
:crossed_fingers:).

Reply:
True

Reply:
Even doe we can do additional fixes and tests on the final v5 when devnet release